internal/sbi/processor/processor.go

```diff
@@ -0,0 +1,32 @@
+package processor
```

Comment:
please refer to https://github.com/free5gc/amf/pull/132/files

You don't need to implement the Consumer getter here.